pkg/virt-api/rest/subresource_test.go

```diff
@@ -99,6 +115,19 @@ var _ = Describe("VirtualMachineInstance Subresources", func() {
 
 	app := SubresourceAPIApp{}
 	BeforeEach(func() {
+		ctrl = gomock.NewController(GinkgoT())
+		kubeClient = fake.NewSimpleClientset()
+		virtClient = kubecli.NewMockKubevirtClient(ctrl)
+		vmClient = kubecli.NewMockVirtualMachineInterface(ctrl)
+		vmiClient = kubecli.NewMockVirtualMachineInstanceInterface(ctrl)
+		migrateClient = kubecli.NewMockVirtualMachineInstanceMigrationInterface(ctrl)
+
+		virtClient.EXPECT().CoreV1().Return(kubeClient.CoreV1()).AnyTimes()
+		virtClient.EXPECT().VirtualMachine(k8smetav1.NamespaceDefault).Return(vmClient).AnyTimes()
+		virtClient.EXPECT().VirtualMachine("").Return(vmClient).AnyTimes()
+		virtClient.EXPECT().VirtualMachineInstance(k8smetav1.NamespaceDefault).Return(vmiClient).AnyTimes()
+		virtClient.EXPECT().VirtualMachineInstance("").Return(vmiClient).AnyTimes()
```

Comment:
same as above.
Why would you expect someone asking for a VMI from an empty namespace?